Integrations/integration-ATD.yml

```diff
@@ -3,12 +3,13 @@ commonfields:
   version: -1
 name: McAfee Advanced Threat Defense
 display: McAfee Advanced Threat Defense
+fromversion: 3.5.0
```

Comment:
Why is this?

Reply:
old version is toversion 3.1.0
this version has no fromversion.